hydra/functions/FlatteLineShape.h

```diff
@@ -218,9 +218,9 @@ namespace hydra {
 
 
                 if(s >= twopimasssq)
-                    rhopipi_real = (2. / 3) * TMath::Sqrt(1 - twopimasssq / s ); // Above pi+pi- threshold
+                    rhopipi_real = (2. / 3) * ::sqrt(1 - twopimasssq / s ); // Above pi+pi- threshold
```

Comment:
This whole piece of code is generating a lot of branching and variable redefinition, which makes execution slow down on gpus. It is also confusing. So deploy Boolean arithmetic here. Example:

instead to write 
```
 if(condition) x = a+b;
else x=a-b;
```
write (1 line !)
```
x = a + (condition==true) *(b) - (condition!=true) *(b) 
```
and so on...
Please, reflect about this.